nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+
+from django.core.management import base
+
+from django_datastream import datastream
+
+
+class Command(base.BaseCommand):
+    help = "Consolidates the latest survey data within the specified time interval \
+           from all nodes and exports the meta graph as a JSON file."
+
+    def add_arguments(self, parser):
+
+        # Optional arguments
+        parser.add_argument(
+            '--datetime',
+            type=str,
+            action='store',
+            dest='datetime',
+            help="All the data will be collected between this UTC datetime \
+                 (format 'yyyy-mm-ddThh:mm:ss' (i.e. '2007-03-04T12:00:00 2007-04-10T12:00:00'), \
+                  and two hours preceding it."
+        )
+
+        parser.add_argument(
+            '-f',
+            '--filename',
+            type=str,
+            action='store',
+            dest='filename',
+            default='survey_export',
+            help="Optional filename",
+        )
+
+        parser.add_argument(
+            '--include-datetime-in-filename',
+            type=bool,
+            action='store',
+            dest='store_datetime',
+            default=False,
+            help="Include datetime in filename?",
+        )
+
+    def handle(self, *args, **options):
+        """
+        Exports the latest survey data graph as a JSON file into the root directory.
+        """
+
+        parsed_datetime = None
+        input_datetime = options['datetime']
+        filename = options['filename']
+
+        if input_datetime:
+            try:
+                parsed_datetime = datetime.datetime.strptime(options['datetime'], "%Y-%m-%dT%H:%M:%S")
+            except ValueError:
+                self.stdout.write(self.style.ERROR("Unable to parse datetime."))
+                return
+
+        exported_graph = extract_survey_graph(parsed_datetime)
+
+        if not exported_graph:
+            self.stdout.write(self.style.ERROR("Insufficient survey data in the datastream during this time period."))
+            return
+
+        if options['store_datetime']:
+            filename = '{0}{1}.json'.format(filename, str(datetime))
+        else:
+            filename = '{0}.json'.format(filename)
+
+        with io.open(filename, 'w', encoding='utf-8') as f:
+            f.write(json.dumps(exported_graph, ensure_ascii=False))
+
+        self.stdout.write(self.style.SUCCESS("Successfully exported survey data."))
+
+
+def extract_survey_graph(parsed_datetime=None):
+    """
+    Extracts a graph of the survey during a specified time period. Only the latest data
+    from specified datetime and two hours preceding the datetime will be used.
+
+    :param parsed_datetime: Datetime object according to the UTC timezone.
+    :return: A dictionary that contains the graph under the 'graph' key.
+    """
+
+    if parsed_datetime:
+        latest_survey_datetime = parsed_datetime
+    else:
+        latest_survey_datetime = datetime.datetime.utcnow()
```

Comment:
I think you should read more about timestamps in Python. You can have timezone aware and non-aware ones. It does not matter how timestamps are stored. Timezones are useful for users, so that they do not have to think in other timezone but can provide their local timezone. We are storing data in a normalized timezone in a database so that it is simpler to store. But when dealing with users it is nice to adapt to what they use.